src/pages/Swap/AdvancedTradingViewBox/AdvancedSwapMode/Chart/ChartLoader.component.tsx

```diff
@@ -5,16 +5,16 @@ import { AbsoluteWrapper } from './Chart.styles'
 
 const THREE_SECONDS = 3000
 
-export function ChartLoader({ symbol }: { symbol?: string }) {
+export function ChartLoader({ pairAddress }: { pairAddress?: string }) {
   const [isLoading, setIsLoading] = useState(true)
 
   useEffect(() => {
     setIsLoading(true)
     const timeout = setTimeout(() => setIsLoading(false), THREE_SECONDS)
     return () => clearTimeout(timeout)
-  }, [symbol])
+  }, [pairAddress])
 
-  return symbol && isLoading ? (
+  return isLoading ? (
```

Comment:
Should we add pairAddress here as well since it might be undefined? So that loader would work properly...Would be good to first merge the toogle pr so that we can test this out properly....

Reply:
Agree, fixed now.

Reply:
Seems that this loader is not working entirely since it takes more time for chart to load...Is this supposed to address it or should we do it in separate pr?
https://user-images.githubusercontent.com/33226956/209844891-cf0ca8f0-f307-4d25-9aca-b9513cc5a7ba.mov

Reply:
For me it works properly. What step did you take to make it work like that?

https://user-images.githubusercontent.com/23079677/209939564-e41ed980-1b99-4433-b1bc-bb984dbc4399.mp4